__tests__/__snapshots__/renderSpec.jsx.snap

```diff
@@ -0,0 +1,257 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is this document something you wrote to go with your tests? Or is it something that was generated by jest? I'm not sure if you meant to commit this because I'm not sure what it's for. If it's intentional, then ignore this comment. :)

Reply:
It is something generated by jest! It's a snapshot that is used by their snapshot testing methods to make sure components are rendering correctly.